src/test/java/edu/ksu/canvas/attendance/services/AttendanceSummaryCSVServiceUTest.java

```diff
@@ -63,9 +66,9 @@ public void createCsvSimpleAttendanceHappyPath() throws IOException {
         boolean isSimpleAttendance = true;
 
         StringBuilder shouldGetThis = new StringBuilder();
-        shouldGetThis.append("\"Name\",\"Total Classes Absent\",\"Total Classes Tardy\"\n");
-        shouldGetThis.append("\""+ARBITRARY_STUDENT_NAME_1 +"\",\""+ARBITRARY_STUDENT_MISSED_1+"\",\""+ARBITRARY_STUDENT_TARDY_1+"\"\n");
-        shouldGetThis.append("\""+ARBITRARY_STUDENT_NAME_2 +"\",\""+ARBITRARY_STUDENT_MISSED_2+"\",\""+ARBITRARY_STUDENT_TARDY_2+"\"\n");
+        shouldGetThis.append("\"Name\",\"Total Classes Absent\",\"Total Classes Tardy\",\"Total Classes Excused\"\n");
+        shouldGetThis.append("\""+ARBITRARY_STUDENT_NAME_1 +"\",\""+ARBITRARY_STUDENT_MISSED_1+"\",\""+ARBITRARY_STUDENT_TARDY_1+"\",\""+ ARBITRARY_STUDENT_EXCUSED_1 +"\"\n");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\",\"" 14 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)